src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -334,12 +334,17 @@ def load_exception(self) -> Dict:
         except Exception:
             return {}
 
-    def load_detail(self) -> Dict[str, list]:
+    def load_detail(self, parse_const_as_str: bool = False) -> Dict[str, list]:
```

Comment:
For what scenario we still need the version that does not handle nan/inf? In another word, can we just remove this flag `parse_const_as_str` and handle nan/inf regardless?

Reply:
before this PR, all NaN/Inf will be float in the memory, and seems work fine, EXCEPT FOR visualize as we need JavaScript to parse it from string; parsing them as a string is kind of a breaking change to me as we change that variable (node/flow output) type from float to string, so I prefer do like this to prevent unexpected customer complain.